README.md

```diff
@@ -104,7 +104,8 @@ static String eWONCertificatePath = "/usr/Certificates";
    },
    "AppConfig":{
       "CycleTimeMs":1000,
-      "PostDuplicateTagValues": false
+      "PostDuplicateTagValues": false,
```

Comment:
This comment is about the image commit (cant comment on it).
-Remove the top bar of your browser to only show the webpage
-Copy the credentials and ip address from the example config file. you dont want to show your real config to the world.
-That image change can probably get merged with this commit

Reply:
I did not think about that.. yup.